e2etest/newe2e_task_runazcopy_parameters.go

```diff
@@ -181,6 +181,7 @@ type GlobalFlags struct {
 	TrustedSuffixes  []string `flag:"trusted-microsoft-suffixes"`
 	SkipVersionCheck *bool    `flag:"skip-version-check"`
 
+	// TODO : Flags default seems to be broken; WI#26954065
```

Comment:
I think this is fixed in another one of my PRs(?) I think in the local RM? I'll double-check when I get back to E2E work.